Fakebook.Profile/Fakebook.Profile.UnitTests/TestData/GenerateRandom.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace Fakebook.Profile.UnitTests.TestData
+{
+    public static class GenerateRandom
```

Comment:
i'm wary of randomness in automated tests, but it's not objectively wrong and there's pros as well